config/config.go

```diff
@@ -25,13 +22,9 @@ type (
 		MorseConfig   *morse.MorseGatewayConfig     `yaml:"morse_config"`
 		ShannonConfig *shannon.ShannonGatewayConfig `yaml:"shannon_config"`
 
-		Services        map[protocol.ServiceID]ServiceConfig `yaml:"services"`
```

Comment:
Not opposed to removing `Services`: letting `EnvoyProxy` handle what service(s) the users can request seems like a cleaner solution.

Do you think a configurable mapping of Authoritative service IDs to QoS implementations makes sense?
e.g.:
```yaml
qos_service_id_mappings:
  "evm":
    - "F001"
    - "F00C"
  "solana":
    - "S001"
    - "SOL"
```

Reply:
To be honest I think it would be better to have one authoritative source of truth for which service uses which QoS. Adding this seems like additional configuration for an unclear purpose.

Is there a use case where an operator would want requests for `Ethereum` to be handled by the QoS service for `Solana`? IMO it's pretty clear cut which QoS is for which service and they are not interoperable.

Even if there does arise a use case wherein interoperable QoS is desired and useful I think we are a long way from there and so I'd recommend not adding this to scope of this PR at this stage.

Reply:
OK, no strong opinions on this from me. If this turns out to be a high-in-demand feature, we will revisit it.